lib/Sema/CSApply.cpp

```diff
@@ -520,6 +520,11 @@ namespace {
     /// a remote distributed actor in the given context.
     bool isDistributedThunk(ConcreteDeclRef ref, Expr *context);
 
+    /// Determine whether the given reference on the given
+    /// base has to be replaced with a distributed thunk instead.
+    bool requiresDistributedThunk(Expr *base, SourceLoc memberLoc,
+                                  ConcreteDeclRef memberRef);
```

Comment:
Cool 👍 I think this will make implementing the known to be local stuff very simple... since we'll add the logic in here to say "well, actually no, we know it is local -> no thunk" ❤️